src/jobs/backup.yml

```diff
@@ -0,0 +1,14 @@
+description: |
+    Generate a backup of production on deployment to staging.
+    `yarn run dxt-backup` 
+
+executor: php
+
+steps:
+    - attach_workspace:
+          at: /home/circleci/project
+
+    - run:
+          name: Run the backup script.
+          command: |
+              yarn run dxt-backup
```

Comment:
Does this verify that the backup finished before the job closes out? 
We wait for this job to finish before we deploy, but we could get into a race condition where

1. We start a deploy
2. This backup job starts
3. It triggers a backup on the WPE platform
4. This backup job ends
5. The deploy process starts
6. Which finishes first? The backup on the WPE platform or the deploy?

Reply:
We _could_ create a cron callback to sniff for the backup to be done before we proceed with the deploy
https://wpengineapi.com/reference#operations-backup-showBackup

Reply:
This trigger's a backup whenever we deploy to staging, and won't fire during a deploy to production as it will have already generated the backup. So basically the backup gets taken of production before you even start doing QA on staging. My thinking was that we still have "confirm backup email" as a step prior to deploying to production.